lib/screens/wallet/components/tokens_cards/components/currency_info_card.dart

```diff
@@ -26,10 +31,17 @@ class CurrencyInfoCard extends StatelessWidget {
       height: cardHeight,
       decoration: BoxDecoration(
         borderRadius: BorderRadius.circular(24),
-        image: DecorationImage(image: AssetImage(tokenBalance.token.backgroundImage), fit: BoxFit.fill),
+        image: DecorationImage(image: tokenBalance.token.backgroundImage, fit: BoxFit.fill),
       ),
       child: Stack(
         children: [
+          if((tokenBalance.token.usecases?.contains('experimental')) ?? false) Container(
+            width: 128,
```

Comment:
Where does the 128 magic number come from?

Reply:
I confess no knowledge of flutter widget layout. All the other children on this widget have hard-coded numerical sizes, so I just picked something that looked good on my emulated screen. Happy to take a suggestion for a better way. @RaulUrtecho

Reply:
It looks like there are about 8 different hard-coded numerical layout constants in here. Do you want me to make them private named constants? Should they be tested against different display screen sizes? Sorry, I am unfamiliar with good practices on this.

Reply:
The truth is that I would leave it as hardcoded, as far as UI is concerned, it is something that can probably be changed for another. To be honest I'm not sure if QA tests on tablets to test the responsive

Reply:
FWIW here is the currency card on a 480x800 screen. I suggest we declare "good enough for now".
![LWCurrency480x800-1](https://user-images.githubusercontent.com/2141014/173483389-601e87e0-dd73-4007-95b8-d34a717e9279.png)

Reply:
Leaving banner graphic at fixed size 128